code/modules/vending/nutrimax.dm

```diff
@@ -22,6 +22,7 @@
 	contraband = list(
 		/obj/item/reagent_containers/cup/bottle/ammonia = 10,
 		/obj/item/reagent_containers/cup/bottle/diethylamine = 5,
+		/obj/item/reagent_containers/cup/bottle/saltpetre = 15,
```

Comment:
> This is massive powercreep but I guess we're all waiting for the day saltpetre is nerfed anyways

No. Saltpetre will be needed more than Diethylamine because of the type of fertilizer rather than power. Diethylamine is needed to stabilize plant and increase yield, Saltpetre is the main fertilizer for potency increase and will be used regardless of its power if it stays the fertilizer focused on potency. 

Also there are 10 ammonia bottles. 2 Diethylamine is 1 Ammonia +1 Ethanol. And Ethanol is essentially free in Biogenerator. So you can turn 10 ammonia bottles into 20 Diethylamine free of charge. Essentially there are 25 bottles of Diethylamine in contraband pool.

Now that you made me realize this, I want to increase the Saltpeter bottle count to be on par with it, to 25 bottles.

And if they see that they have just 5 bottles which translates to 150 units - they will think that this is not enough, and there is no point in hacking machine and spending credits to buy contraband (which is twice more expensive btw). It will make them go for that free chem dispenser board anyway, which invalidates this PR.

If Chem Dispenser is an essential machine for hydroponics, just place it there roundstart. This PR at least tries to make them play the designed way.

Reply:
It's an issue of mechanics. Many don't understand how nutrients work and that E-Z nutriment is just fine to boost potency from 0 to 100 even in dirt. But this PR is not about skill issues and design of nutrients. 

It's just about chem dispenser not being stolen roundstart just to record one recipe and stay idle 99% of time.